mis_builder/tests/test_aep.py

```diff
@@ -352,11 +352,11 @@ def test_get_aml_domain_for_expr(self):
                 # debi[700IN]
                 '&',
                 ('account_id', 'in', (self.account_in.id, )),
-                ('debit', '>', 0),
+                ('debit', '<>', 0),
                 # crdi[400AR]
                 '&',
                 ('account_id', 'in', (self.account_ar.id, )),
-                ('credit', '>', 0),
+                ('credit', '<>', 0),
```

Comment:
Why is it necessary to change the test?

Reply:
At first look it seemed that test were failing... Now, i belive it is not necessary to change tests, but if we change module logic, what harm is to adapt tests to it?

Reply:
Since the change is backwards compatible, this PR must not change tests, to prove there is no regression. (adding new tests is not practical in this case)

Reply:
agree... done